maestro/lifecycle.py

```diff
@@ -108,12 +110,21 @@ def _create_env(self):
         return dict((str(k), str(v)) for k, v in env.items())
 
     def _test(self, container=None):
-        return subprocess.call(self.command, env=self._create_env()) == 0
+        env = self._create_env()
+        if self.envfrom == "env":
```

Comment:
Use single quotes here too for consistency with the coding style.

Reply:
done